File: 코딩테스트 입문/Javascript/수 조작하기2.js
function solution(numLog) {
    let answer = "";
    
    numLog.forEach((v,i)=>{
        if(i>0){
            if(v-numLog[i-1] === 1){
                answer += "w";
            }
            else if(v-numLog[i-1] === -1){
                answer += "s";
            }
            else if(v-numLog[i-1] === 10){
                answer += "d";
            }
            else if(v-numLog[i-1] === -10){
                answer += "a";
            }
        }
    })
    
    return answer;
}

File: 코딩테스트 입문/Python/수 조작하기2.py
def solution(numLog):
    answer = ''
    data = 0  
    
    for index,value in enumerate(numLog):
        if index>=1:
            if value-data == 1:
                answer += "w"
            elif value-data == -1:
                answer += "s"
            elif value-data == 10:
                answer += "d"
            elif value-data == -10:
                answer += "a"
        data = value
    
    return answer